index.bs

```diff
@@ -2078,14 +2081,10 @@ dictionary IDBVersionChangeEventInit : EventInit {
 };
 </xmp>
 
-The <dfn attribute for=IDBVersionChangeEvent>oldVersion</dfn>
-attribute getter returns the previous version of the database.
+The <dfn attribute for=IDBVersionChangeEvent>oldVersion</dfn> getter steps are to return the value it was initialized to. It represents the previous version of the database.
```

Comment:
Ideally this concept of "value it was initialized to" would be replaced with explicit associated values, but this is an improvement over the previous version.

Reply:
Yeah... I went looking for how other specs deal with event interfaces and they seem to have not gotten the modernization other places have. Leaving this for now.

Reply:
Oh, I missed that this was an event interface. Yeah, then this seems good, for sure.